### ISSUE TITLE
Module Menu "Start Level / End Level" not working

### ISSUE THREAD

**Issue by shoulders** [joomla/joomla-cms#27171]
### Steps to reproduce the issue
Set up a multilevel menu
![image](https://user-images.githubusercontent.com/319997/69754034-d495df80-114c-11ea-8715-4d2d09d2bc02.png)
Create a menu module with a modified start level to remove the 1st level menu item
![image](https://user-images.githubusercontent.com/319997/69754108-f7c08f00-114c-11ea-9d9c-49735355424f.png)

### Expected result
The first level menu should be removed. In my case it was an alias so I can use the secondary menu but maintain the URL structure.
![image](https://user-images.githubusercontent.com/319997/69754164-20488900-114d-11ea-9bc6-b10c129fb1d8.png)

### Actual result

Nothing. the menu actually dissapears

### System information (as much as possible)

PHP Built On | Linux xxxxxxxxxxxxxxxxxxx 3.10.0-962.3.2.lve1.5.26.7.el7.x86_64 #1 SMP Wed Oct 2 07:53:12 EDT 2019 x86_64
-- | --
Database Type | mysql
Database Version | 5.5.5-10.3.18-MariaDB
Database Collation | utf8_general_ci
Database Connection Collation | utf8mb4_general_ci
PHP Version | 7.3.11
Web Server | Apache
WebServer to PHP Interface | fpm-fcgi
Joomla! Version | Joomla! 3.9.13 Stable [ Amani ] 5-November-2019 15:00 GMT
Joomla! Platform Version | Joomla Platform 13.1.0 Stable [ Curiosity ] 24-Apr-2013 00:00 GMT
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:70.0) Gecko/20100101 Firefox/70.0

### Additional comments
This has been an issue since Joomla 2.5 and has been previously reported on the old tracker

http://joomlacode.org/gf/project/joomla/tracker/?action=TrackerItemEdit&tracker_id=8103&tracker_item_id=27190

I discovered some solutions to this isssue on a joomla forum https://forum.joomla.org/viewtopic.php?t=687090

A workaround has been provided by **toxicman666** back in **2012** in modules/mod_menu/helper.php line 69 he commented out the line
````php
|| ($start > 1 && !in_array($item->tree[$start - 2], $path)))
````
Which works for me. there is also another solution made 2 years later which does not work. where abother guy has modified the if statement. I think it is just the last line again that has been altered by **cMagnafico**
````php
if (($start && $start > $item->level)
    || ($end && $item->level > $end)
    || (!$showAll && $item->level > 1 && !in_array($item->parent_id, $path))
    || ($start>1 && !in_array($path[0],$item->tree)) )
````

Tar

**Comment by brianteeman** [comment]
I believe the problem is that you have chosen "current" and an alias is not the current. What happens if you select the real menu item as the bas

**Comment by shoulders** [comment]
@brianteeman sorry for the delay I had to take a chimney down.

## Module menu parent as a real menu
I had checked to see if using an alias as a parent affected this but I did not put that in the ticket above. I have redone the menu but using a single article as the parent in the module menu and this arrangement still does not work. The whole menu dissapears.
![image](https://user-images.githubusercontent.com/319997/69898763-72f49180-1355-11ea-8d57-528438fa4efa.png)

![image](https://user-images.githubusercontent.com/319997/69898759-5a847700-1355-11ea-9fce-d506ab6d2abd.png)

## Using the real menu item of the alias as a base
I dont understand what this function is for (possibly menu visibility?). I 
![image](https://user-images.githubusercontent.com/319997/69898837-4d1bbc80-1356-11ea-9634-c9440244b5cf.png)
![image](https://user-images.githubusercontent.com/319997/69898850-5dcc3280-1356-11ea-833c-4e4a82629f0a.png)

This arrangement does not work either. I have also tried any combination of start level and end level and unless start level is 1 the menu will not display

**Comment by brianteeman** [comment]
Thank you for raising this issue.

Joomla 3 is now in security only mode with no further bug fixes or new features.

This issue has been resolved in Joomla 4 with #38563  and it will now been closed.

If we are mistaken and this does apply to Joomla 4 please open a new issue (and reference this one if you wish) with updated details for testing in Joomla 4.
cc [@zero-24](https://github.com/zero-24)
